### DISCUSSION TITLE
What Stream operators do you wanted to be provided out of box.

### DISCUSSION THREAD

**Discussion by He-Pin** [apache/pekko#605] [General]
Hi , community, I want to open up this discussion for what operators you want pekko to provided out of box, I find  both FS2 and ZIO has a rich set of operators.

**Comment by pjfanning** [comment]
We definitely want to add more stream operators. It is good to discuss them first because we don't want to have an excessively large number of operators either. If anyone is thinking of porting fs2, zio or rxjava operators to pekko, that is great - the licenses are compatible.

Anyone thinking of copying Akka 2.7/2.8 operators, please stay clear of Pekko. The licenses are not compatible. We will need people to declare that they have looked at Akka 2.7/2.8 code before submitting PRs to Pekko.

**Reply by pjfanning** [reply]
https://github.com/apache/incubator-pekko/issues/624 is first issue opened related to this.
